tests/testflows/rbac/requirements/requirements.py

```diff
@@ -0,0 +1,6132 @@
+# These requirements were auto generated
+# from software requirements specification (SRS)
+# document by TestFlows v1.6.200629.1155222.
+# Do not edit by hand but re-generate instead
+# using 'tfs requirements generate' command.
+from testflows.core import Requirement
+
+RQ_SRS_006_RBAC = Requirement(
```

Comment:
What does the `006` mean?

Reply:
This our internal SRS `id`. In this case the `id` for this SRS is `SRS-006`.

Reply:
I mean it's just a number from `requirements.md`, it doesn't mean anything special, isn't it?

Reply:
Yes, it just a number.